modular_skyrat/master_files/code/modules/clothing/under/jobs/engineering.dm

```diff
@@ -0,0 +1,38 @@
+/obj/item/clothing/under/rank/engineering
+	worn_icon_digi = 'modular_skyrat/master_files/icons/mob/clothing/under/engineering_digi.dmi' //Anything that was in the engineering.dmi, should be in the engineering_digi.dmi
+
+/obj/item/clothing/under/rank/engineering/engineer/skyrat
+	icon = 'modular_skyrat/master_files/icons/obj/clothing/under/engineering.dmi'
+	worn_icon = 'modular_skyrat/master_files/icons/mob/clothing/under/engineering.dmi'
+
+//Add a /obj/item/clothing/under/rank/engineering/chief_engineer/skyrat or /obj/item/clothing/under/rank/engineering/atmospheric_technician/skyrat if you add uniforms for either
+
+//////////////////////////////////////
+////////////// ENGINEER //////////////
```

Comment:
```suggestion
/*
ENGINEER
*/
```

could lead to autodoc weirdness

Reply:
woe is I
I've not seen it cause issues but better safe than sorry; I'll hit all the other ones I added in the other cleanup PRs too

Reply:
wait, no, it doesnt, TG uses it a lot already for similar sorting (see: packs.dm)